website/content/docs/overview.md

```
@@ -0,0 +1,98 @@
+simplifies configuration compared to manually setting up each VPN tunnel and
+mapping it to a VLAN on a traditional router. With OpenPERouter, the
+configuration is managed using Kubernetes Custom Resource Definitions (CRDs),
+allowing you to declaratively define VPN tunnels and their properties.
+
+## A separate network namespace
+
+The router runs in a separate network namespace, and interacts with the host
+using a veth pair serving as entry points for the overlays. This allows it to
+be seen as a completely external component from a networking point of view.
+
+![](/images/openpeinside.svg)
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)